qiskit_sphinx_theme/furo/base/static/styles/qiskit_changes.css

```diff
@@ -5,13 +5,14 @@
  * Variables that start with `--qiskit-` are new values we use in this file; they are not overrides.
  */
 body {
-    --color-brand-primary: #8A3FFC;
-    --color-brand-content: #8A3FFC;
+    --color-brand-primary: var(--qiskit-color-purple);
+    --color-brand-content: var(--qiskit-color-purple);
```

Comment:
do we use the `--color-brand-primary` and `--color-brand-content` vars anywhere? If they are the same colour anyway do we need both?

Reply:
They are furo variables used in a whole bunch of places. It's the main way in Furo that you override colors.